src/main/java/com/example/authservice/messaging/MQConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.example.authservice.messaging;
+
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MQConfig {
+
+    public static final String QUEUE = "message_queue";
+    public static final String EXCHANGE = "message_exchange";
+    public static final String ROUTING_KEY = "message_exchange";
```

Comment:
A descriptive or unique exchange name will be helpful for identifying this exchange amongst other exchanges and perhaps a routing key with a similar format to that in both the handouts and Martin's example is also a good idea.
https://github.com/kappsegla/JWTAuthWithGatewayExample/commit/e346474fbe50871657f6a302ce22c2cc7727cd87